scylla-gdb.py

```diff
@@ -1307,6 +1310,10 @@ def matches_symbol(name):
         return matches_symbol
 
     def __call__(self, name):
+        name = name.strip()
+        if re.search(self._coro_pattern, name) is not None:
```

Comment:
nit: ` is not None` is not required.
`if re.search(self._coro_pattern, name):` would work just as well, is shorter, and straight forward.